### PR TITLE
[#27] Implemented sample usage for the lib.

### PR DESCRIPTION
**Pull request by tamerharsh**

### CODE REVIEW

File: sample/sample.cpp [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#ifndef SAMPLE_CPP
+#define SAMPLE_CPP
+
+#include <iostream>
+#include "logger/logger_lib.hpp"
+
+
+void CreateLoggerObject()
+{
+logger::Logger logging_obj ;
```
Comment (sachinsnale) [inline sample/sample.cpp]: Please look after proper indentation,
please check for whole file

**Review by sachinsnale — APPROVED** [review]
Just do that indentation changes

File: sample/sample.cpp [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  logger::SET_LOGGER_NAME("new logger");
+  logger::LOG(logger::log_level::DEBUG, "sample message");
+}
+
+bool SampleDivideFunction(int number_1, int number_2) 
+{ 
+  if (number_2 == 0) {
+    throw std::exception("Divisor cannot be zero.");
+     } else
+    return number_1 / number_2;
```
Comment (sachinsnale) [inline sample/sample.cpp]: Either have this
```
if (number_2 == 0)
    throw std::exception("Divisor cannot be zero.");
else
    return number_1 / number_2;
```
Or this
```
if (number_2 == 0){
    throw std::exception("Divisor cannot be zero.");
}
else{
    return number_1 / number_2;
}
```

**Review by sachinsnale — CHANGES_REQUESTED** [review]

**Review by sachinsnale — APPROVED** [review]

**Comment by tamerharsh** [comment]
Merging with approval.

**Pull request merged**